.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,38 +6,45 @@ labels: enhancement
 assignees: ""
 ---
 
-### Description
+<!--
+
+⚠ Please note that this project is released with a Contributor Code of Conduct.
+By participating in this project you agree to abide by its terms.
+Read more: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/blob/master/CODE_OF_CONDUCT.md
+
+All sections are just template. Remove if they don't help your case.
+
+-->
+
+## Description
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4743215)